notify-send.scm

```diff
@@ -0,0 +1,177 @@
+(define-module (notify-send)
```

Comment:
I don't like the module name.
Maybe `(notify send)` or just add it to `(notify)` ?